src/main/java/org/spongepowered/api/item/inventory/generation/ItemStackBuilderPopulators.java

```diff
@@ -0,0 +1,197 @@
+        checkNotNull(level, "VariableAmount cannot be null!");
+        checkNotNull(enchantment, "Enchantment cannot be null!");
+        return enchantments(VariableAmount.fixed(1), ImmutableList.of(new Tuple<>(enchantment, level)));
+    }
+
+    public static BiConsumer<ItemStack.Builder, Random> enchantmentsWithDefaultVariance(Collection<Enchantment> enchantments) {
+        return enchantmentsWithDefaultVariance(VariableAmount.fixed(1), ImmutableList.copyOf(enchantments));
+    }
+
+    public static BiConsumer<ItemStack.Builder, Random> enchantmentsWithDefaultVariance(VariableAmount amount, Enchantment enchantment, Enchantment... enchantments) {
+        return enchantmentsWithDefaultVariance(amount, ImmutableList.<Enchantment>builder().add(enchantment).addAll(Arrays.asList(enchantments)).build());
+    }
+
+    public static BiConsumer<ItemStack.Builder, Random> enchantmentsWithDefaultVariance(VariableAmount amount, Collection<Enchantment> itemEnchantments) {
+        checkNotNull(amount, "Variable amount cannot be null!");
+        checkNotNull(itemEnchantments, "Enchantment collection cannot be null!");
+        return enchantments(amount, () -> {
+            return itemEnchantments.stream()
+                    .map(enchantment -> {
+                        checkNotNull(enchantment, "Enchantment cannot be null!");
+                        final int minimum = enchantment.getMinimumLevel();
+                        final int maximum = enchantment.getMaximumLevel();
+                        return new Tuple<>(enchantment, VariableAmount.baseWithRandomAddition(minimum, maximum - minimum));
+                    })
+                    .collect(Collectors.toList());
+        });
+    }
+
+    public static BiConsumer<ItemStack.Builder, Random> enchantments(VariableAmount amount, Collection<Tuple<Enchantment, VariableAmount>> enchantments) {
+        return enchantments(amount, () -> enchantments);
+
+    }
+
+    public static BiConsumer<ItemStack.Builder, Random> enchantments(VariableAmount amount, Supplier<Collection<Tuple<Enchantment, VariableAmount>>> enchantments) {
+        checkNotNull(enchantments, "Supplier cannot be null!");
+        checkNotNull(amount, "VariableAmount cannot be null!");
+        return (builder, random) -> {
+            final ItemStack temp = builder.build();
+            final EnchantmentData enchantmentData = temp.getOrCreate(EnchantmentData.class).get();
+            ListValue<ItemEnchantment> enchantmentListValue = enchantmentData.enchantments();
+            final ImmutableList<Tuple<Enchantment, VariableAmount>> list = ImmutableList.copyOf(enchantments.get());
+            final int rollCount = amount.getFlooredAmount(random);
+            checkArgument(rollCount >= 1, "Must have at least 1 enchantment!");
+            WeightedTable<Tuple<Enchantment, VariableAmount>> table = new WeightedTable<>(rollCount);
```

Comment:
This will be moved outside of the `BiConsumer` when a new `WeightedTable` can be constructed with a `VariableAmount` for rolls.